### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by dstad418**
Completed HW #3.

To run, go into the project directory and type: npm install

The dependencies should load, and then you can hit: npm start

To create it on your local machine at localhost:3000.

### CODE REVIEW

File: hw3/styles.css [hunk 1]
```diff
@@ -7,13 +8,71 @@
...
+}
+
+.character:hover {
+  background-color: var(--blue);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  color: var(--white);
+}
+
+.character img {
+  width: 100%;
```
Comment (lebenpdx) [inline hw3/styles.css]: This seems to be causing the last image to fill in the entire width due to an "uneven" number of characters. My fix for this was to set an amount of pixels the width should be instead.

File: hw3/02-charts.js [hunk 2]
```diff
@@ -26,27 +28,93 @@ const borderColors = [
...
+          data,
           backgroundColor: backgroundColors,
           borderColor: borderColors,
           borderWidth: 1,
         },
       ],
     },
+    options: {
+      legend: {
+        display: false
```
Comment (lebenpdx) [inline hw3/02-charts.js]: I understand not wanting a legend in some cases, however with so many different groups, it becomes hard to read interpret the data at a glance. I saw you mention that you do have not seen game of thrones, but I think some grouping of the 1 person groups could be helpful.

**Review by lebenpdx — COMMENTED** [review]
Parts 1 and 2 look good for the most part. I had some minor suggestions. I could not manage to get part 3 to run on my end, however.

**Review by SayleeMJ — COMMENTED** [review]
The code looks very good, but I would recommend a  few minor changes in the part 1 and 2:

For Part 1, I recommend adjusting the image height to 300px or another appropriate size to ensure similarity.

Regarding Part 2, I suggest displaying labels to make it easier to read and understand.

**Review by SayleeMJ — COMMENTED** [review]
I was not able to run part 03 (project) on my device

**Comment by dstad418** [comment]
Completed changes per reviews!

**Comment by mattsloe** [comment]
Code Styling
* AirBnB guide requires parentheses around arrow function arguments
  - instead of `arg => {}`, use `(arg) => {}`
* Remember to use single quotes for strings in JS

Requirements
* Ex 01 - Formatting is good but not exactly like the requirements. (4 per line on larger viewports and 2 per row on smller viewport.)
* I like your comments in the .css file, they help understand the thinking behind certain decisions (like adding transition)
* Good job on filtering the houses data for Ex 02

I wasn't able to check the third exercise, I think there were dependency issues on my end.

Great work!

**Pull request merged**